### DISCUSSION TITLE
Flat_csv Driver properties .

### DISCUSSION THREAD

**Discussion by BogdanMugerli** [dbeaver/dbeaver#12495] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version: Windows 10
- DBeaver version 21.0.3.202104181339
- Additional extensions:   DBeaver Office Support	1.1.91.202104042040

#### Connection specification:
- Database name and version: Flat_csv
- Driver name: Flat files (CSV) / CSV
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:
Go to Connection settings / Driver properties.
If you leave quotechar blank, after reconnect the value of the separator is copied into quotechar.

#### Steps to reproduce, if exist:
Go to Connection settings / Driver properties.
Set separator to character { and quotechar to blank.
Close Dbeaver, 
Open Dbeaver.
Look into Flat_csv driver properties.
Quotechar has value {.

**Comment by uslss** [comment]
@BogdanMugerli cannot reproduce an issue. You could probably provide a video with an issue or any other info that could help investigate it?

**Comment by BogdanMugerli** [comment, marked as answer]
Hi. After further investigation seems like Driver for csv cannot have empty quotechar.
[CSV quotechar.pdf](https://github.com/dbeaver/dbeaver/files/6419694/CSV.quotechar.pdf)
